vnext/Microsoft.ReactNative/Views/FrameworkElementViewManager.cpp

```diff
@@ -699,4 +712,37 @@ void FrameworkElementViewManager::RefreshTransformMatrix(ShadowNodeBase *shadowN
   }
 }
 
+void FrameworkElementViewManager::ApplyAccessibility(winrt::Windows::UI::Xaml::FrameworkElement const& element,
+  winrt::Windows::UI::Xaml::Automation::Peers::AccessibilityView const& value) {
+
+  xaml::Automation::AutomationProperties::SetAccessibilityView(element, value);
+  
+  int childrenCount = winrt::VisualTreeHelper::GetChildrenCount(element);
+  //iterate through all of the children. 
+  for (int i = 0; i < childrenCount; i++){
```

Comment:
Could you walk the shadow node hierarchy instead of the XAML hierarchy?

Reply:
E.g.:
```
void FrameworkElementViewManager::ApplyAccessibility(ShadowNodeBase *node, xaml::Automation::Peers::AccessibilityView const &value) {
  if (const auto uiManager = GetNativeUIManager(m_content).lock()) {
    xaml::Automation::AutomationProperties::SetAccessibilityView(node->GetView(), value);
    for (const auto childTag : node->m_children) {
      const auto childNode = uiManager->getHost()->findNode(childTag);
      ApplyAccessibility(childNode, value);
    }
  }
}
```

Reply:
This implementation actually ended up not going anywhere because when we're trying to iterate through the children here in FEVM the children don't actually exist yet. I haven't worked with the shadow node hierarchy much, do you think that would help here regarding that? 

Another implementation I'm working on is based off of what Garrison does and that's to add an automation peer for our ViewPanel which will be invalidated as a whole when no-hide-descendants is set. I have some local code for that from a while back, I'll open up a new draft PR to rehydrate that work since we had another likely related issue filed recently.